package.json

```diff
@@ -13,7 +13,7 @@
     "build:test": "cp envConfig/test.env .env && npm run webpack:prod",
     "build:live": "cp envConfig/live.env .env && npm run webpack:prod",
     "cypress:proxy": "export NO_PROXY=localhost,localhost.bbc.com && cypress",
-    "cypress": "npm run cypress:proxy -- run",
+    "cypress": "npm run cypress:proxy -- run | perl -pe 's/\\x1b\\[[0-9;]*m//g'",
```

Comment:
This means we dont have colour locally :( can you add this perl thing onto a make command please, so it only affects CI